.devcontainer/docker-compose.yaml

```
@@ -8,6 +8,7 @@ services:
   # Custom strapi service
   strapi:
     # Build the strapi service from its folder
+    container_name: strapi
```

Just to ensure that we do not get local name conflicts
```suggestion
    container_name: clic-strapi
```